impl/pom.xml

```diff
@@ -144,9 +144,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <configuration>
-                    <includeDependencySources>true</includeDependencySources>
+                    <includeDependencySources>false</includeDependencySources>
```

Comment:
Since this is "bundle" jar where binary contains API as well as implementation, it is expected that dependency sources are included (= `true`). Other than that, at least on my environment:
```
ljungman-mac:jsonp lukas$ mvn -version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /Users/lukas/java/apache-maven-3.6.3
Java version: 11.0.5, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-11.0.5.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```
running `mvn -U -C clean install -Poss-release -Dnon.final=true` works fine without changing this.

What do I have to do in order to reproduce the problem you are trying to fix?

Reply:
I believe he is trying to fix the error in our builds when the Maven "deploy" target is enabled.

See the console output from build 28: https://ci.eclipse.org/jsonp/job/jsonp-master-build/28/console

> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:jar (attach-javadocs) on project jakarta.json: MavenReportException: Error while generating Javadoc: /home/jenkins/agent/workspace/jsonp-master-build/impl/target/apidocs/src/org.glassfish.java.json -> [Help 1]

Reply:
thanks for the pointer. If I got it right, then the problem is not here but in the job set up. Result of execution of `mvn clean install deploy` is different from `mvn clean deploy` - see https://ci.eclipse.org/jsonp/job/jsonp-master-build/30 which fails on the missing jaxb api (...this is also the reason I asked for providing exact command & environment details)

to make it short - no, this particular change is wrong, does not resolve the problem and should not be done

Reply:
IK, this change was reverted.